CHANGELOG.md

```diff
@@ -1,10 +1,32 @@
+2.0.0 (In Progress) - Release
+===================
+
+Breaking API Changes
+--------------------
+
+### Active call no longer accessible through the Device object.
+
+`Device.activeCall` is no longer available. Instead,
+the application should keep track of active calls itself, i.e. when accepting an
+incoming call or when making an outgoing call.
+
+```ts
+const token = '...';
+const options = { ... };
+const device = new Device(token, options);
+
+await device.register();
+
+const activeCall = await device.connect();
+```
+
 2.0.0 (Apr 30, 2021) - Pilot
```

Comment:
I would expect a Pilot release to have a specifier at the end to indicate as much. Did it have a preview or beta suffix?

Reply:
It did not, but it was also never released to `npm`, only a GitHub release tag.